### ISSUE TITLE
SIM910 only applies to dictionary literals

### ISSUE THREAD

**Issue by ThiefMaster** [astral-sh/ruff#8760]
This doesn't trigger a violation even when using the example from the [docs](https://docs.astral.sh/ruff/rules/dict-get-with-none-default/#example).

```python
ages = {"Tom": 23, "Maria": 23, "Dog": 11}
age = ages.get("Cat", None)  # None
```

```
$ ruff --isolated --select SIM910 --preview --no-cache rufftest/ruff_sample.py
(no output)
```

(ruff 0.1.6)

**Comment by charliermarsh** [comment]
It looks like it only applies to dictionary literals right now (like `{}.get(...)`). We can improve that.
